CMakeLists.txt

```diff
@@ -20,11 +20,17 @@ include(GNUInstallDirs)
 set_property(GLOBAL PROPERTY TARGET_MESSAGES OFF)
 
 if (NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Build mode ('Debug' or 'Release', default is 'Debug')" FORCE)
+  set(CMAKE_BUILD_TYPE "DebugSan" CACHE STRING "Build mode ('DebugSan' or 'Debug' or 'Release', default is 'DebugSan')" FORCE)
```

Comment:
I'm not sure if debug + sanitizer is the best default. Compiling with sanitizers takes longer and slows down development. I would assume that most users use dev builds by default and only enable sanitizer if needed, at least that's what I do most of the time.

However, having a dev+san build always running in the CI is a good idea and should catch bugs before they are integrated into `development`.

Reply:
That's a good point actually. I'll change that slightly to have plain `Debug` as default.